benchmarking/commons/utils.py

```diff
@@ -142,3 +143,14 @@ def find_or_create_requirements_txt(
     else:
         fname = files[0]
     return fname
+
+
+def get_master_random_seed(random_seed: Optional[int]) -> int:
+    if random_seed is None:
+        random_seed = np.random.randint(0, 2**32)
```

Comment:
NIT: Python runtimes have maxint of `2**63-1` ([long renamed as int](https://peps.python.org/pep-0237/)) so its fine but if we want to be compatible with standard 32bit int, I think we should use `[0-2**31) == [0-maxint + 1)`

Reply:
Hmm, but I think the random seed range of numpy.random.RandomState is only 32 bit.

BTW: RandomState has been replaced by something else recently in Python, but a first glance looked a bit complex to me, so I left it. Feel free to look into this, I am not bound to RandomState, just we should use the same thing all over the place.

Reply:
Good point, I think random state is indeen unsigned int so 0-2**32 is great.